<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="styleSheet">
   <string notr="true">QPushButton:flat {
    border: none;
	background: transparent;
	padding: 3px;
}
QPushButton:flat:hover {
    border: 1px solid lightgray;
}
QPushButton:flat:checked {
	border: 1px solid #01bccd;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
}

QToolBar {
    spacing: 3px;
}

QTableView::item:selected:!active {
    background: qlineargradient(spread:reflect, x1:0.539, y1:0, x2:0.489, y2:1, stop:0 rgba(150, 150, 0, 100), stop:1 rgba(200, 200, 0, 100));
	color: black;
}
QTableView::item:selected:active {
    background: qlineargradient(spread:reflect, x1:0.539, y1:0, x2:0.489, y2:1, stop:0 rgba(150, 150, 0, 200), stop:1 rgba(200, 200, 0, 200));
	color: black;
}
QTableView::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #EAEAEA, stop: 1 #DDDDDD);
}
QTableView::focus {
	background-color: rgb(255, 255, 255);
    border: 2px solid #ffff00;
}
QTableView::!focus {
	background-color: rgb(240, 240, 240);
	border: 1px solid rgb(0, 0, 0);
}

QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
 	color: white;
    padding: 6px;
    border: 1px solid #6c6c6c;
}

QHeaderView::section:checked:vertical
{
	background-color: white;
	color: black;
}

QListView {
	background: lightgray;
	color: black;
	border: 1px solid black;
}
QListView::item:selected:active, QListView::item:selected:!active {
/*
    background: qlineargradient(spread:reflect, x1:0.539, y1:0, x2:0.489, y2:1, stop:0 rgba(125, 125, 0, 200), stop:1 rgba(175, 175, 0, 200));
*/	
	background: darkgray;
	color: white;
}
QListView::item, QListWidget::item {
	padding: 4px;
}

QTabWidget::pane { /* The tab widget frame */
    border: 2px solid #C2C7CB;
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 1px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 6px;
    border-bottom-left-radius: 6px;
    min-width: 6ex;
    min-height: 30ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-left-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-left: 3px; /* make non-selected tabs look smaller */
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame {
background: gray;
border: 1px solid black;
margin: 0;
padding: 0;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="furqanButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>WhatsThisCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
background: none;
color: white;
border: 0;
margin: 0;
padding: 0;
}

QPushButton:hover {
background: none;
color: yellow;
border: 0;
margin: 0;
padding: 0;
}</string>
        </property>
        <property name="text">
         <string>Furqan</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minimizeButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/minimize</normaloff>:/minimize</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/exit</normaloff>:/exit</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,100">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame {
background: gray;
color: white;
border: 1px solid black;
margin: 0;
padding: 0;
show-decoration-selected: 1;
}

QListView::item:selected:active, QListView::item:selected:!active {
	background:white;
	color: black;
}

QListView::item:hover {
/*
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #EAEBEC, stop: 1 #CCCCCC);
	color: gray;

QListView::item:hover, QListWidget::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #EAEBEC, stop: 1 #CCCCCC);
	color: gray;
	padding: 0;
}

*/	
	border: 1px solid lightgray;
}

QListView::item {
	padding: 4px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QFrame" name="controlFrame">
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: lightgray;
	border: 1px solid black;
	padding: 1px;
}
QPushButton {
	padding: 3px;
	color: gray;
}
QPushButton:hover {
	color: white;
	border: 1px solid gray;
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>&amp;Add</string>
            </property>
            <property name="icon">
             <iconset resource="../resource.qrc">
              <normaloff>:/table-add</normaloff>:/table-add</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>&amp;Modify</string>
            </property>
            <property name="icon">
             <iconset resource="../resource.qrc">
              <normaloff>:/table-modify</normaloff>:/table-modify</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>&amp;Remove</string>
            </property>
            <property name="icon">
             <iconset resource="../resource.qrc">
              <normaloff>:/table-remove</normaloff>:/table-remove</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="printHorizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="printButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>&amp;Print</string>
            </property>
            <property name="icon">
             <iconset resource="../resource.qrc">
              <normaloff>:/table-print</normaloff>:/table-print</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minimizeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>670</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>704</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
